app/controllers/patients_controller.rb

```diff
@@ -364,7 +368,8 @@ def update_fields(patient, params)
       # NOTE: In the case where a patient is being moved back to the exposure workflow, the symptom onset should be overwritten
       #       because if a case is being ruled out (moved back to exposure), that patient no longer has no known symptom onset and
       #       shouldn't immediately be put back on the symptomatic line list unless they have symptomatic reports.
-      reset_symptom_onset(params_to_update, params, patient)
+      params_to_update.concat(%i[user_defined_symptom_onset symptom_onset])
+      reset_symptom_onset(params[:patient], patient, :system)
 
       # Set extended isolation to nil.
       params_to_update << :extended_isolation
```

Comment:
So I still think this case is not handled: 
> If system sets extended isolation to nil when moving from isolation to exposure

Because I would expect a history item to have to be logged here. I tested and did not see it as well.

Reply:
addressed in https://github.com/SaraAlert/SaraAlert/pull/371/commits/8b0a671188f7f9777ae1f19ec1056bc5be7a4751#diff-4f4c29154577c3b2ecf2075ade80b28eR378